src/components/Cards/CardDemo.scss

```diff
@@ -0,0 +1,29 @@
+@import './../../styles/fonts.scss';
+@import './../../styles/utilities/breakpoints.scss';
+
+/* stylelint-disable */
+.cards--large {
+  padding: 30px;
+  background-color: #f2edf8;
+}
+
+.cards--small {
+  padding: 30px;
+}
+
+.cards--square {
+  padding: 30px;
+  background-color: #f2edf8;
+}
+
+.card-large {
+  margin: 10px;
+}
+
+.card-small {
+  margin: 10px;
+}
+
+.card-square {
+  margin: 10px;
+}
```

Comment:
Please add a space to the last line. I had this issue too but was able to set a config for it in the editor.

Reply:
@mcheng09 , fixed this in latest push